payments/card/direct.md

```diff
@@ -43,11 +46,15 @@ https://www.pcisecuritystandards.org/)." %}
   with gathered `Purchase` information.
 * The action taken next is the `direct-authorization` operation that is returned
   in the first request. You `POST` the payer's card data to the URL in
-  the[`direct-authorization` operation][authorization].
-* If 3-D Secure authentication is required, you will then receive a URL where
-  you will have to redirect the payer.
-* When the payment is completed, the payer needs to be redirected back to your
-  merchant/webshop site.
+  the [`direct-authorization`][authorization] operation.
+* If the issuer requires 3-D Secure authentication, you will then receive an
+  operation called `redirect-authentication`. You must redirect the payer to this
+  URL to let them authenticate against the issuers 3-D secure page.
+* When the 3-D Secure flow is completed, the payer needs to be redirected
+  back to your merchant/webshop site.
+* If the issuer does not require 3-D Secure authentication, the payment will
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[unordered-list-marker-style] Marker style should be `-`](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5311783)